index.mjs

```diff
@@ -331,13 +365,14 @@ function diff3Merge(a, o, b, options) {
     stringSeparator: /\s+/
   };
   options = Object.assign(defaults, options);
+  options.timeout = getAssertTimeout(options.timeout);
```

Comment:
I think we shouldn't assign this back to the options object
generally I think it is confusing to overwrite things with a different type.